src/netex-convertor/testdata/multiServicesPeriodData.ts

```diff
@@ -0,0 +1,27 @@
+const multiServicesPeriodData = {
```

Comment:
Does this one need its own testdata file or could it live in test-data.ts?

Reply:
Have now moved multiServicesPeriodData into test-data.